README.md

```diff
@@ -1,2 +1,9 @@
 Email Tracker
 =============
+
+Simple project to tracker my personal email. They'll not tell you *"Sure, I read your email"*.
```

Comment:
It's not wrong, but you could say: "The tracker won't say for sure if the user read the email. It can only confirm if the email was open or not."